grails-app/services/io/xh/toolbox/App/NewsService.groovy

```diff
@@ -0,0 +1,88 @@
+package io.xh.toolbox.App
+
+import io.xh.hoist.json.JSON
+import io.xh.toolbox.BaseService
+import io.xh.toolbox.NewsItem
+import org.grails.web.json.JSONArray
+import org.grails.web.json.JSONObject
+
+import static io.xh.hoist.util.DateTimeUtils.MINUTES
+
+class NewsService extends BaseService {
+
+    public List<NewsItem> _newsItems
+
+    static clearCachesConfigs = ['newsSources', 'newsApiKey']
+    def configService
+
+    void init() {
+        log.info('news service initialized')
+        createTimer(
+                runFn: this.&loadAllNews,
+                interval: 10 * MINUTES
```

Comment:
This would be a nice place to use a feature of the `Timer` class where we can pass a string for the interval and have it read from a soft-config.  Our standard would define a config like `newsRefreshMins` where the config is an int representing the frequency in minutes.  That name (string) can then be passed here as the interval, along with an `intervalUnits: MINUTES` param to specify that the config is in minutes vs ms.

Although just tried to run locally and saw that we were rate limited out! I guess if PIQ is deployed and running it might be stealing hits from us - let's see if we can stop that....